code/controllers/subsystem/persistence/_persistence.dm

```diff
@@ -62,6 +64,7 @@ SUBSYSTEM_DEF(persistence)
 	load_delamination_counter()
 	load_tram_counter()
 	load_adventures()
+	load_circuits()
```

Comment:
I don't think we need all circuits from all ckeys loaded immediately. We should load on demand for the specific ckey.